packages/dev/core/src/Gizmos/boundingBoxGizmo.ts

```diff
@@ -633,6 +635,17 @@ export class BoundingBoxGizmo extends Gizmo implements IBoundingBoxGizmo {
                 m.isVisible = !selectedMesh || m == selectedMesh;
             });
     }
+
+    protected _unhoverMeshOnTouchUp(pointerId: number, selectedMesh: AbstractMesh) {
+        const engine = this.gizmoLayer.originalScene.getEngine();
+        const isMobileSafari = engine._badOS || (engine._badDesktopOS && document && "ontouchend" in document);

```

Comment:
I'm pretty sure we have that it somewhere. I'll search later

Reply:
mobile safari hack no longer needed as of 16309d8